lib/local-cluster/lib/Cardano/Wallet/Launch/Cluster/Monitoring/Http/Application/Server.hs

```diff
@@ -0,0 +1,103 @@
+{-# LANGUAGE RecordWildCards #-}
+
+module Cardano.Wallet.Launch.Cluster.Monitoring.Http.Application.Server
```

Comment:
```suggestion
module Cardano.Wallet.Launch.Cluster.Http.Faucet.Server
```

This place in the module hierarchy would make more sense to me? 🤔 

I would definitely remove the `Monitoring` prefix.

I don't mind `Application`, though the term is a bit generic. `Faucet` would be more specific.